docs/en/docs/lakehouse/multi-catalog/es.md

```diff
@@ -68,27 +68,28 @@ After switching to the ES Catalog, you will be in the `dafault_db`  so you don't
-| ------------- | ----------- | ------- |
-| null          | null        |         |
-| boolean       | boolean     |         |
-| byte          | tinyint     |         |
-| short         | smallint    |         |
-| integer       | int         |         |
-| long          | bigint      |         |
-| unsigned_long | largeint    |         |
-| float         | float       |         |
-| half_float    | float       |         |
-| double        | double      |         |
-| scaled_float  | double      |         |
-| date          | date        |         |
-| keyword       | string      |         |
-| text          | string      |         |
-| ip            | string      |         |
-| nested        | string      |         |
-| object        | string      |         |
-| other         | unsupported |         |
-
+| ES Type       | Doris Type  | Comment                                                                 |
+| ------------- | ----------- |-------------------------------------------------------------------------|
+| null          | null        |                                                                         |
+| boolean       | boolean     |                                                                         |
+| byte          | tinyint     |                                                                         |
+| short         | smallint    |                                                                         |
+| integer       | int         |                                                                         |
+| long          | bigint      |                                                                         |
+| unsigned_long | largeint    |                                                                         |
+| float         | float       |                                                                         |
+| half_float    | float       |                                                                         |
+| double        | double      |                                                                         |
+| scaled_float  | double      |                                                                         |
+| date          | date        | Only support default/yyyy-MM-dd HH:mm:ss/yyyy-MM-dd/epoch_millis format |
+| keyword       | string      |                                                                         |
+| text          | string      |                                                                         |
+| ip            | string      |                                                                         |
+| nested        | string      |                                                                         |
+| object        | string      |                                                                         |
+| other         | unsupported |                                                                         |
+
+<version since="dev">
```

Comment:
`dev`?
Why not  `1.2.3`?

Reply:
Currently is dev, modify it when cherry pick it to version branch.